src/__tests__/pages/CreateNhsAppTemplate.test.tsx

```diff
@@ -20,15 +21,15 @@ test('renders page', async () => {
   );
   expect(container.asFragment()).toMatchSnapshot();
 
-  const templateNameBox = document.getElementById('nhsAppTemplateName');
+  const templateNameBox = document.querySelector('#nhsAppTemplateName');
   if (!templateNameBox) {
-    fail('Template name box not found');
```

Comment:
can't use `fail` it comes from jest/jasmine which is slowly being depreciated . The suggestion is to throw an error instead.